tests/unit/wait-for-promise-test.ts

```diff
@@ -2,6 +2,7 @@ import { module, test } from 'qunit';
 import { Promise } from 'rsvp';
 import { _reset, waitForPromise, getPendingWaiterState } from 'ember-test-waiters';
 import MockStableError, { resetError, overrideError } from './utils/mock-stable-error';
+import EmberObject from '@ember/object';
 import { DEBUG } from '@glimmer/env';
 
 if (DEBUG) {
```

Comment:
Whoops, now I see where this pattern came from. I do prefer to do this inside the module call though.

Reply:
Ya I think I can take credit for doing it that way.